Schedule Checkout

nishimweelysee opened the pull request:
#### What does this PR do?
> This Task Checkout the users from accommodation according to their checkout date.
>
#### Description of Task to be completed
> - The system should be able to checkout users whose checkout date is current date
> - number of people the facility can accommodate increased after the scheduled checkout.
#### How should this be manually tested?
> -  cron the branch `ch-scheduleCheckout`
> - run `npm install`
> - run `npm run start`
> - book accommodation
> - wait fro that date of accommodation checkout to be reached

#### Any background context you want to provide?
 this Tasks uses node-cron which helps to schedule tasks.
#### What are the relevant Trello story?
> Chore (Scheduled checkout) use this Trello link to view the story https://trello.com/c/a7s8vaHz
#### Screenshots (if appropriate)
>
>

nishimweelysee committed:
Chore to Schedule Checkout users from accomodation:
 - When date reaches the user checkedout
 - The room status beacome inactive
 - User get Notification

muhireheir reviewed (commented):
I've booked and set checkout date in future after that i changed the date in my pc but idint see the change (number of roomsReft did'nt change)

nishimweelysee commented:
> 
> 
> I've booked and set checkout date in future after that i changed the date in my pc but idint see the change (number of roomsReft did'nt change)

Corrected
